kannon/master.py

```diff
@@ -10,34 +10,41 @@
 from .task import TaskOnBullet
 from .kube_util import create_job, JobStatus, gen_job_name, get_job_status
 
-
 logger = logging.getLogger(__name__)
 
 
 class Kannon:
+
     def __init__(
         self,
         api_instance: client.BatchV1Api,
         namespace: str,
         image_name: str,
         container_name: str,
-        service_account_name: str,
         job_prefix: str,
-        path_child_script: str,
-        env_to_inherit: List[str],
+        service_account_name: str,
+        persistent_volume: str = None,
+        persistent_volume_claim: str = None,
+        path_child_script: str = "./run_child.py",
+        env_to_inherit: List[str] = ["TASK_WORKSPACE_DIRECTORY"],
```

Comment:
I think there are too many args, and other k8s resource will add more and more args.
You should separate args for kannon and for k8s.
And then try some way to separate them.

e.x)

```python
class Kannon:
    def __init__(self, 
                         # kannon resource
                         path_child_script: str = "./run_child.py",
                         env_to_inherit: List[str] = ["TASK_WORKSPACE_DIRECTORY"], # Or, it should be set by user in `template_job`? I don't sure.
                         # k8s resource
                         template_job: kubernetes.batch.v1.Job, 
```

Reply:
I think some libraries like Kannon will helpful. 

* [kubeflow](https://www.kubeflow.org/)
* [hera-workflow](https://github.com/argoproj-labs/hera-workflows)
* [prefect](https://www.prefect.io/)

Reply:
Thank you for the idea,

> template_job: kubernetes.batch.v1.Job

I agree with this style, too. Commit 15ebf87 supported it.

I'd prefer to enable `kannon` lib to support multiple ways child template `template_job` is created, i.e., it can be created from a yaml file `child-template.yaml` or by the python API.